docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -1,45 +1,81 @@
+fast finality, and
+energy efficiency. For the whitepaper, see [here](https://www.avalabs.org/whitepapers). 
+
+Key Features Include: 
+
+- Speed: Avalanche consensus provides sub-second, immutable finality, ensuring that transactions are
+quickly confirmed and irreversible.
+- Scalability: Capable of a peak throughput of up to 50,000 transactions per second with a latency
+of less than half a second.
+- Energy Efficiency: Unlike other popular consensus protocols, participation in Avalanche consensus
+is not computationally intensive nor expensive. 
+- Adaptive Security: Avalanche consensus is designed to resist various attacks, including sybil
+attacks, distributed denial-of-service (DDoS) attacks, and collusion attacks. Its probabilistic
+nature ensures that the consensus outcome converges to the desired state, even when the network
+is under attack. 
 
 ![Consensus Comparison](/img/Consensus-protocol-comparison.png)
 
-Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+## Conceptual Overview
+
+Consensus protocols in the Avalanche family operate through repeated sub-sampled voting. When a
 [validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
 determining whether a
 [transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
-accepted, it asks a small, random subset of validators on their preferences. Each queried validator
-replies with the transaction that it prefers, or thinks should be accepted. The transaction that is
-received from the reply might be the same transaction indicating agreement, or it might be another
-conflicting transaction.
+accepted, it asks a small, random subset of validators for their preference. Each queried validator
+replies with the transaction that it prefers, or thinks should be accepted.
+
+:::note
+
+Consensus will never include a transaction that is determined to be **invalid**. For example, if you
+were to submit a transaction to send 100 AVAX to a friend, but your wallet only has 2 AVAX, this transaction
+is considered **invalid** and will not participate in consensus. 
+
+:::
+
+If a majority of the validators sampled reply with the same preferred
+transaction, this becomes the preferred choice of the group sampled, including the validator that asked.
+In the future, this validator and all those who have participated in at least one round of sampling
+will reply with the transaction preferred by the majority.
 
-If a sufficiently large portion (_alpha_ α) of the validators sampled reply with the same preferred
-transaction, the validator adopts the preferred transaction of that majority. That is, when it is
-queried about its preference in the future, it replies that it thinks this transaction the majority
-preferred should be accepted instead.
+Overtime, more and more validators are queried, and the group in agreement grows larger and larger.
```

Comment:
I'd omit this line. It's kind of begging the question. i.e. it's explaining how the consensus protocol works by saying "everyone comes to consensus"

Reply:
addressed